Kernel/Arch/i386/CPU.cpp

```diff
@@ -1233,8 +1237,12 @@ UNMAP_AFTER_INIT void Processor::initialize(u32 cpu)
     if (cpu == 0) {
         VERIFY((FlatPtr(&s_clean_fpu_state) & 0xF) == 0);
         asm volatile("fninit");
-        asm volatile("fxsave %0"
-                     : "=m"(s_clean_fpu_state));
+        if (has_feature(CPUFeature::FXSR))
+            asm volatile("fxsave %0"
+                         : "=m"(s_clean_fpu_state));
+        else
+            asm volatile("fnsave %0"
+                         : "=m"(s_clean_fpu_state));
```

Comment:
Please wrap the `if` and `else` blocks in `{ }` since they span multiple lines. :)

Reply:
The linter is not happy about that. I guess it doesn't want braces on what is technically a single statement.

Reply:
That's weird? `clang-format` is (somewhat unfortunately) not opinionated about braces AFAIK.

But okay fine.